packages/cli/src/tasks/environment.ts

```diff
@@ -21,6 +40,7 @@ export const createConfig: ListrTask<ICreateConfig> = {
 export type IClearConfig = { configDir: string }
 export const clearConfig: ListrTask<IClearConfig> = {
   title: 'Clear config',
+  skip: ctx => hasOtherInstances(ctx.configDir),
```

Comment:
this system only works if every CLI is started in the same folder because the default value of `ctx.configDir`.
I think the default value should be a folder in the `HOME` directory.